### ISSUE TITLE
temp file holding currently run seeds may lead to conflicts if multiple users run seeds at once?

### ISSUE THREAD

**Issue by mwvoss** [HBClab/RestingState#80]
### Summary
within seedCorrelation/<nuisanceRegression>/rois/ there is a text file called seeds.txt that lists the currently running or last run seeds. The purpose of the issue is to pose what happens if two people are running a batch of seeds at the same time?

### Expected behavior
We want two people to be able to run seeds at once without overwriting each-others seed list

### How to replicate the behavior
run a new seed and watch the previous seed list get replaced

### Script/Workflow details

Please put URL to code or code here (if not too long).

**Comment by jdkent** [comment]
As long as people use docker this should not be an issue anymore, agree @mwvoss?

**Comment by tbweng** [comment]
> As long as people use docker this should not be an issue anymore, agree @mwvoss?

I believe this would still be an issue since the network drive is being mapped to docker with `-v`. I'm not well versed in the mechanics of how docker mounts volumes, but I think the scripts would still read/write to the drive. One solution could be to copy `seeds.txt` to a tmp dir within the docker container.

**Comment by mwvoss** [comment]
it seems like if two people start a batch of subjects at the same time, with different seeds, the `seeds.txt` could still change before docker mounts and loads the info into a more protected memory.

**Comment by jdkent** [comment]
Yeah, you both are correct, I wasn't thinking of the right file, sorry.

I think my current recommendation for sanity's sake is to initialize the data directory as a git repository (or perhaps datalad repository eventually), then clone it locally and run whatever seeds you want, and if you have something you would like to share, you can push it back to the server.

There may be a programmatic solution too, like user-stamping and/or time-stamping that file and selecting the "correct" one in subsequent runs.

**Comment by mwvoss** [comment]
Thanks @jdkent...it'd be great if we can brainstorm simpler solutions than using github or datalad, more along the lines of the second set of options..

**Comment by tbweng** [comment]
> There may be a programmatic solution too, like user-stamping and/or time-stamping that file and selecting the "correct" one in subsequent runs.

Proposed solution:
`seedlist=seed.txt; newSeedlist=$(basename ${filename%.txt}).$(whoami).$(date "+%Y.%m.%d-%H.%M.%S").txt`

1. set `newSeedlist` variable with unique stamps at beginning of run.
2. reference the variable throughout the script.
3. `rm "${newSeedlist}"`?
